pages/common/docker.md

```diff
@@ -4,36 +4,36 @@
 
 - List currently running docker containers:
 
-`docker container ls`
+`docker ps`
```

Comment:
Was this meant to be `ls` instead of `ps`?

Reply:
Nope, that's the actual command. I assume it's one of the many reasons why they did the CLI restructure, adding in the newer sub-commands like `container`, `image`, etc.

Reply:
I see. If the `container` commands are newer, why are we changing the page to the older commands? I'm confused.

Reply:
@sbrl, see #2592. Looks like the short commands were going to be removed, but then most people tend to use the short commands. It looks like @MasterOdin was thinking of using additional pages for the long commands. So `docker-container`, etc.

Reply:
Ah, I see. This change is ok then :-)

Maybe once those pages are done we can revisit and add a `See also`.

Reply:
Yeah, it's kind of a weird spot Docker is in. They added a bunch of sub-commands (container, image, etc.) to allow adding a lot more commands for those specific actions, but left the top level largely alone. As such, while all of the top level commands do have an equivalent action within a sub-command, most docs and things really just use these handful of key top level commands still.